ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -421,3 +421,59 @@ presubmits:
         name: ""
         resources: {}
     trigger: (?m)^/test yaml-lint
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/launch-aws
+    decorate: true
+    name: pull-ci-openshift-installer-master-launch-aws
+    rerun_command: /test launch-aws
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/launch-aws-cluster-profile
+        - --target=launch-aws
+        - --template=/usr/local/launch-aws
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-installer-master.yaml
+              name: ci-operator-configs
+        - name: JOB_NAME_SAFE
+          value: launch-aws
+        - name: TEST_COMMAND
+          value: sleep 6900
```

Comment:
Do we need to sleep?  If we just want to exercise the `setup` and `teardown` containers, couldn't this command just be `exit 0`?

Reply:
the point of launch is to hold the cluster open so you can use it, that's why it's sleeping

Reply:
basically you run `/test launch-aws` and then you have a cluster you can use for X minutes.

Reply:
Yeah, launch is a fairly general convention we have.  It'll be part of the general CI docs.